docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -251,4 +256,6 @@ Do not use untagged or “latest”.
 			- Any available cost or benchmarking information. 
 			- How to cite the use of your workflow (including references for the original software authors). 
 
+- Note: Documentation can be housed at either or both, the metadata section of the workflow file and/or the GitHub README document. On Dockstore, if a description is provided in the metadata section, it will be displayed on the INFO tab. If the metadata section is missing, Dockstore will display the README on the INFO tab.
```

Comment:
Wording sounds a little off. I think it'd sound better as `Documentation can be housed in the metadata section of the workflow file and/or the GitHub README document.`